client/src/assets/messages/landing.ts

```diff
@@ -1,34 +1,75 @@
 // Strings for the landing page.
 
-import { Titles } from '@/assets/messages/common';
-
 /**
  * Strings related to the scorecard-related part of the landing page.
  */
 export class ScorecardSearch {
-  static HEADER = 'Get your drinking water score';
-  static BODY = `Understand the likelihood of your home or community's exposure to lead service lines.`;
-  static CTA_PLACEHOLDER = 'Enter your address';
-  static CTA_BUTTON = 'Get Score';
+  static HEADER = 'How Safe is My Water?';
```

Comment:
Can you make these variables more specific to what they are? We want the content writers to edit this in github directly and I think it'll be easier for them to find things

Reply:
Throughout the file

Reply:
This isn't possible with the current setup which requires an interface to be passed into the general section component. As discussed offline it may be better to remove the shared component since styling + format of messages differs across sections.

Reply:
Removed use of interface and added more specific names.